plugins/woocommerce/client/admin/client/task-lists/fills/Marketing/Plugin.tsx

```diff
@@ -64,6 +90,7 @@ export const Plugin: React.FC< PluginProps > = ( {
 							{ __( 'Built by WooCommerce', 'woocommerce' ) }
 						</Pill>
 					) }
+					{ getPills( tags ) }
```

Comment:
No need to fix the UI for now.  I think we should tweak the UI if we will show more than 1 tags in the future.

<img width="630" alt="Screenshot 2025-02-24 at 15 21 50" src="https://github.com/user-attachments/assets/6dca92d0-d0f3-4d80-b937-2e48700a92cd" />

Reply:
Did the UI get fixed? I'm just seeing the one pill now (which would be ideal) 🤔

Reply:
My thinking is that we'll only have space to display one pill at a time, but I suppose it is possible for a "Built by WooCommerce" extension to also be available on marketplace in future, creating the need for two pills. An alternative to the marketplace pill might be to put "marketplace" into the button text instead 🤔 

<img width="695" alt="Screen Shot 2025-02-24 at 6 20 34 pm" src="https://github.com/user-attachments/assets/1149aff2-d11e-4f41-b658-e2285fceaec9" />

Reply:
> I guess this was just a mockup

Yeah, that was a mockup!

Reply:
I think it's a good follow-up, however, I feel to only implement when it becomes necessary. Is that ok?

Reply:
Sounds good!